tests/test_AnoGam.py

```diff
@@ -17,35 +17,46 @@ def test_name(self):
     def test_common_name(self):
         assert self.species.common_name == "Anopheles gambiae"
 
-    # QC Tests. These tests are performed by another contributor
-    # independently referring to the citations provided in the
-    # species definition, filling in the appropriate values
-    # and deleting the pytest "skip" annotations.
-    @pytest.mark.skip("Population size QC not done yet")
     def test_qc_population_size(self):
-        assert self.species.population_size == -1
+        # based on theta = 4 Ne u since diversity in most pops is 1.5%
+        assert pytest.approx(self.species.population_size, 0.015 / 3.5e-9)
```

Comment:
I'm very confused on this, because the code says the pop size of 6e6 is from the "Ghana population", but there's very few references to Ghana in the paper (one colony used for 'calibration and validation' is from Ghana, but there didn't seem to be demographic inference done on that one?). This number is what I would have put in, based on nucleotide diversity?

Reply:
Ugh typo here-- this should be "Gabon" not "Ghana" 

As for population size- I have no clue where that number came from now. I must have gotten it from somewhere... that nucleotide diversity number above is approx the average across species- should we just pin everything on the Gabon sample / demographic history? That is the GAS model I implemented already.

Reply:
Computing from theta = 4 Ne u and the species mutation rate for Gabon sounds good to me!

Reply:
actually 0.015 looks about correct for \pi. this is from the Phase 2 paper: 
![image](https://user-images.githubusercontent.com/7674197/114631396-57033e80-9c71-11eb-8fc6-631176126f65.png)

Reply:
That gets an answer of 1,000,000 if I did my factors of 4 right.

Reply:
perhaps you can double-check?

Reply:
(andy checked this)